### PR TITLE
Basic functional tests

### PR DESCRIPTION
**Pull request by juandopazo**
This just checks if the site is responding or not. I can't think of anything else worth doing that doesn't require PhantomJS or a browser. And I don't want to walk that path at the moment. Any ideas? /cc @tripp

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: tests/functional/routes.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    [
+        '/',
+        '/about/',
+        '/guide/',
+        '/integrations/',
+        '/github/',
+        '/handlebars/',
+        '/react/',
+        '/dust/'
+    ].forEach(function (route) {
```
Comment (ericf) [inline tests/functional/routes.js]: It would be nice to get this list from the Express app at some point…

**Comment by ericf** [comment]
@juandopazo were these tests suppose to run in Travis?

**Comment by ericf** [comment]
Also can the README be updated about running the unit and functional tests?

**Comment by juandopazo** [comment]
@ericf if you think that's worth it, I can give it a try. I've been experimenting with using https://www.npmjs.org/package/grunt-run to start and stop the site and it should work.

**Comment by ericf** [comment]
> @ericf if you think that's worth it, I can give it a try

If it's straight forward I think it will be good to do. Right now I'm not sure how you'd run these tests even locally.

**Comment by juandopazo** [comment]
@ericf ok how do you like that?

File: package.json [hunk 2]
```diff
@@ -52,6 +52,9 @@
     "grunt-broccoli-build": "^0.4.0",
     "grunt-cli": "^0.1.13",
     "grunt-contrib-clean": "^0.6.0",
+    "grunt-mocha-istanbul": "~2.3.0",
+    "grunt-run": "~0.3.0",
+    "grunt-simple-mocha": "~0.4.0",
```
Comment (okuryu) [inline package.json]: I think that we can use `^` instead of `~`.

Comment (juandopazo) [inline package.json]: Oh. I just used `npm i --save-dev`. I thought it was using `^`. I'll change it.

Comment (ericf) [inline package.json]: What npm version do you have?

**Comment by juandopazo** [comment]
I guess you liked it :shipit:

**Pull request merged**